packages/web/src/scss/components/Dropdown/_Dropdown.scss

```diff
@@ -1,6 +1,9 @@
 @use 'theme' as theme;
 
 .Dropdown {
```

Comment:
Hmm, I know that you're not the author of previous code lines, but... Don't we want to apply these styles (except `display: none`) when the dropdown is open?